standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions.toml

```diff
@@ -0,0 +1,138 @@
+[[dimensions]]
+class = "County"
+type = "geography"
+name = "ACS County 2018"
+file = "dimensions/counties.csv"
+description = """
+American Community Survey US counties, 2018.
+"""
+
+[[dimensions]]
+class = "Sector"
+type = "sector"
+name = "TEMPO LDV BEV Charging"
+file = "dimensions/tempo.csv"
+description = """
+TEMPO provides LDV BEV charging profiles.
+"""
+
+[[dimensions]]
+class = "Subsector"
+type = "subsector"
+name = "TEMPO Bin"
+file = "dimensions/bin.csv"
+description = """
+TEMPO demographic/geographic/vehicle bins.
+
+This is a cross-join of the following supplemental dimensions:
+* Composition
+* Income
+* Urbanity
+* Class
+* Tech
+"""
+
+# [[dimensions]]
+# type = "subsector"
+# name = "Composition"
+# file = "dimensions/supplemental/composition.csv"
+# description = """
+# Household composition.
+# """
+# 
+# [[dimensions]]
+# type = "subsector"
+# name = "Income"
+# file = "dimensions/supplemental/income.csv"
+# description = """
+# Income range.
+# """
+# 
+# [[dimensions]]
+# type = "subsector"
+# name = "Urbanity"
+# file = "dimensions/supplemental/urbanity.csv"
+# description = """
+# Urban classifications, according to [Claritas definitions](https://healthpolicy.ucla.edu/chis/data/public-use-data-file/Documents/2003PUF_constructs_c.pdf#page=24).
+# """
+# 
+# [[dimensions]]
+# type = "subsector"
+# name = "Class"
+# file = "dimensions/supplemental/class.csv"
+# description = """
+# Passenger vehicle class.
+# """
+# 
+# [[dimensions]]
+# type = "subsector"
+# name = "Tech"
+# file = "dimensions/supplemental/tech.csv"
+# description = """
+# Vehicle technology.
+# """
+
+[[dimensions]]
+class = "EnergyEndUse"
+type = "metric"
+name = "TEMPO LDV BEV Usage"
+file = "dimensions/end_use.csv"
+description = """
+Fuels used by TEMPO BEVs.
+"""
+
+[[dimensions]]
+class = "Time"
+type = "time"
+name = "Time of Week by Month"
+#start = "Jan Mon 00:00:00"
+#end   = "Dec Sun 23:59:59"
+ranges = [{start = "Apr Mon 00:00:00", end = "Apr Sun 23:00:00"}]
```

Comment:
What happens to this? Does an arbitrary date get assigned? (Sorry, I think I am overly stuck on the fact that the docs said the data needs to be "in UTC". Maybe that's just not right?)

Reply:
If we make a datetime from this the year will be something invalid like 1900, but we won't use it. This is only for validating the contents of the load data.